osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/scheduling/OsgpScheduler.java

```diff
@@ -109,18 +110,28 @@ public JobDetail createJobDetail(final Class<? extends Job> jobClass) {
    *     OsgpScheduler#createJobDetail(Class)}.
    * @param cronExpression A CRON-expression.
    * @param timeZone A Timezone for the CRON-expression.
+   * @param jobDataMap A map containing data that can be used by job.
    * @return A {@link Trigger} instance.
    */
   public Trigger createJobTrigger(
```

Comment:
This method is only used locally. Can be private, I think

Reply:
No we use it from SMHE